dev/com.ibm.ws.channelfw/src/com/ibm/ws/tcpchannel/internal/WorkQueueManager.java

```diff
@@ -200,7 +205,7 @@ protected void startSelectors(boolean inBound) throws ChannelException {
                 // add
                 connectCount = new int[maxChannelSelectorsPerFlow]; // 269309 add
 
-                connect[0] = new ConnectChannelSelector(this, 0, CS_CONNECTOR);
+                connect[0] = new ConnectChannelSelector(this, 0, CS_CONNECTOR, startImmediately);
                 createNewThread(connect[0], CS_CONNECTOR, 1);
 
                 for (int i = 0; i < maxChannelSelectorsPerFlow; i++) {
```

Comment:
should line 439 - `CS[nextOpen] = new ConnectChannelSelector(this, nextOpen, CS_CONNECTOR)` - pass `startImmediately` as well?

Reply:
The constructor in line 439 is triggered when all active selectors are full. The change being proposed by this PR is during the scope of the server's startup. It is unlikely that there will be enough traffic during startup to need to need to create additional selectors.

Reply:
As a connect selector, line 439 should be outbound, so passing the config value directly. For the SocketRWChannelSelector, checking if its an oubtound read/write before passing the configuration value.